src/sitemap/en-slugs.json

```diff
@@ -3940,7 +3940,7 @@
     "id": "Q902",
     "importedAt": "2020-07-16T11:05:19.496Z"
   },
-  "List_of_presidents_of_the_United_States_who_owned_slaves": {
+  "List_of_presidents_of_the_United_States_who_owned_politicians": {
```

Comment:
This sentence, at least to me, doesn't make sense. politicians !== slaves.

Reply:
I makes sense to me https://en.wikipedia.org/wiki/List_of_presidents_of_the_United_States_who_owned_slaves